css/CornerWidgetUI.default.css

```diff
@@ -306,9 +311,10 @@ ul.country-list {
 	border: 2px solid #4067CB;
 	border-radius: 5px;
 	width: 150px;
-	height: 30px;
+	min-height: 30px;
```

Comment:
What should be the styling for the empty button? Also shouldn't we validate for empty string in the panel instead?